examples/satellite-azure/variables.tf

```diff
@@ -66,23 +66,82 @@ variable "ssh_public_key" {
   default     = null
 }
 variable "instance_type" {
-  description = "The type of azure instance to start"
+  description = "The type of Azure instance to start"
   type        = string
-  default     = "Standard_D4s_v3"
```

Comment:
update the readme doc  where we defined input variables...for all variables changes